x-pack/plugins/cross_cluster_replication/public/app/services/auto_follow_pattern_validators.js

```diff
@@ -38,14 +38,14 @@ export const validateName = (name = '') => {
     if (name[0] === '_') {
       errorMsg = i18n.translate(
         'xpack.crossClusterReplication.autoFollowPattern.nameValidation.errorUnderscore',
-        { defaultMessage: "Name can't begin with an underscore." }
+        { defaultMessage: 'Name can\'t begin with an underscore.' }
```

Comment:
Aren't we using `avoid-escape`?

Reply:
Looks like we aren't - will take that up in another issue.

Reply:
👍 I'll fix the instances introduced by this PR anyway.